src/geoarrow.cpp

```diff
@@ -338,6 +340,10 @@ void init_geoarrow(py::module& m) {
             can be a ``pyarrow.DataType`` or ``pyarrow.Field``, and you can
             use the ``geoarrow.pyarrow`` package to construct such geoarrow
             extension types.
+        projection : spherely.Projection, default Projection.lnglat()
```

Comment:
```suggestion
        projection : :py:class:`Projection`, default :py:meth:`Projection.lnglat`
```

Although maybe both are rendered as hyperlinks by sphinx? Either way it needs `Projection` be added to the API reference doc.

Reply:
So I added it to the API docs to see if the linking would go automatically, which I suppose will be the case>? (at least for Geography it works) 

But our docs get build with the released s2geography, of course, so this is not yet included

Reply:
Yes let's figure out and address this later if needed when next s2geography version is released.